lib/paths.cf

```diff
@@ -570,6 +570,29 @@ bundle common paths
 
       "path[shadow]"       string => "/etc/security/passwd";
 
+@if minimum_version(3.16)
```

Comment:
@nickanderson this section and addition of root_path and prefix_path as usable elsewhere in policy is starting to feel like "too much" focused on termux issues. Should these differences have to be handled by folks each time they write policy for termux?

Reply:
its nice if it just works for people, not having to be super explicit if doing so doesn't add a bunch of overhead for other platforms.